ci-operator/step-registry/ipi/conf/openstack/ipi-conf-openstack-commands.sh

```diff
@@ -13,4 +13,9 @@ UNSAFE_CLUSTER_NAME=${NAMESPACE}-${JOB_NAME_HASH}
 SAFE_CLUSTER_NAME=${UNSAFE_CLUSTER_NAME#"ci-op-"}
 echo "${SAFE_CLUSTER_NAME}" > ${SHARED_DIR}/CLUSTER_NAME
 
+if [ -f "/var/run/cluster-secrets/${CLUSTER_TYPE}/clouds.yaml" ]; then
+  cp "/var/run/cluster-secrets/${CLUSTER_TYPE}/clouds.yaml"  "${SHARED_DIR}/clouds.yaml"
+fi
 
+echo "${OPENSTACK_EXTERNAL_NETWORK}" > ${SHARED_DIR}/OPENSTACK_EXTERNAL_NETWORK
+echo "${OPENSTACK_COMPUTE_FLAVOR}" > ${SHARED_DIR}/OPENSTACK_INSTANCE_FLAVOR
```

Comment:
These environment variables are set at a workflow level and to my understanding, are exposed to all steps. Why persisting them in files?

Reply:
To setup for cluster_type change. Once this patch merges. we can switch the rest of workflows. 
As a matter of fact this patch is getting too large. Need to refactor into smaller pieces againg